cmd/dfget/app/root.go

```diff
@@ -184,9 +193,11 @@ func initFlags() {
 	flagSet.StringVarP(&cfg.Output, "output", "o", "",
 		"Destination path which is used to store the requested downloading file. It must contain detailed directory and specific filename, for example, '/tmp/file.mp4'")
 
-	// localLimit & totalLimit & timeout
+	// localLimit & minRate & totalLimit & timeout
 	flagSet.StringVarP(&localLimit, "locallimit", "s", "",
 		"network bandwidth rate limit for single download task, in format of 20M/m/K/k")
+	flagSet.StringVarP(&minRate, "minrate", "x", "",
```

Comment:
It's unnecessary to set short flag for `minRate`.

Reply:
Handled.